pow/randomx/sys/build.rs

```diff
@@ -17,7 +17,11 @@ fn main() {
 	let dst = config.build();
 
 	if target.contains("pc-windows-msvc") {
-		println!("cargo:rustc-link-search=native={}/build/Release", dst.display());
+		if env::var("PROFILE").unwrap() == "debug" {
```

Comment:
```suggestion
		if env::var("PROFILE").expect("PROFILE env not found") == "debug" {
```

Reply:
Right, have been deviated by reading this file and cmake crate where it`s only used unwrap 😞